packages/math-input/src/components/keypad/keypad-mathquill.stories.tsx

```diff
@@ -78,7 +78,7 @@ export function V2KeypadWithMathquill() {
                             basicRelations
                             divisionKey
                             logarithms
-                            multiplicationDot
+                            convertDotToTimes
```

Comment:
The three variable names sounded confusing as they were: `convertDotToTimes`, `multiplicationDot`, and `times`. multiplicationDot seems like it would technically be the inverse of the original variable logic (times), so I've renamed things in order to maintain readability.